src/common/card/Card.tsx

```diff
@@ -0,0 +1,158 @@
+
+const Button = (props: { children: React.ReactNode }) => (
+  <button className={classnames(styles.footerButton, 'btn', 'btn-link')}>
+    {props.children}
+  </button>
+);
+
+const EditButton = (props: { children: React.ReactNode }) => (
+  <button className={classnames(styles.editButton, 'btn', 'btn-link')}>
+    {props.children}
+  </button>
+);
+
+const HorizontalDelimiter = () => <hr className={styles.hruler} />;
+
+const VerticalDelimiter = () => <hr className={styles.vdivider} />;
+
+const Delimiter = () => (
+  <div className="flex-fill">
+    <VerticalDelimiter />
+    <HorizontalDelimiter />
+  </div>
+);
+
+const ContentTable = (props: { children: React.ReactNode }) => (
+  <table className="table table-borderless">
+    <tbody>{props.children}</tbody>
+  </table>
+);
+
+const Card = ({ width, Header }: { width: string; Header: React.FC }) => (
+  <div style={{ width }}>
+    <Container fluid={true} className={styles.wrapper}>
+      <Header />
+      <Row>
+        <Col md={{ span: 4 }}>
+          <>
+            <EditButton>Edit</EditButton>
+            <h1>Venepaikan vaihto</h1>
+            <article>
+              <ContentTable>
+                <tr>
+                  <td>Jono nro:</td>
+                  <td>245</td>
+                </tr>
+                <tr>
+                  <td>Kirjattu</td>
+                  <td>23.8.2019 kello 21.06</td>
+                </tr>
+                <tr>
+                  <td>Status:</td>
+                  <td>Ei käsitelty</td>
+                </tr>
+              </ContentTable>
+            </article>
+          </>
+          <>
+            <h1>Muita tietoja</h1>
+          </>
+        </Col>
+
+        <Delimiter />
+        <Col md={{ span: 4 }}>
+          <>
+            <h1>Purjevene / Moottoripursi</h1>
+            <article>
+              <ContentTable>
+                <tr>
+                  <td>Pituus:</td>
+                  <td>8,7m</td>
+                </tr>
+                <tr>
+                  <td>Leveys:</td>
+                  <td>3,2m</td>
+                </tr>
+                <tr>
+                  <td>Syvyys:</td>
+                  <td>-</td>
+                </tr>
+                <tr>
+                  <td>Paino:</td>
+                  <td>-</td>
+                </tr>
+              </ContentTable>
+            </article>
+          </>
+          <>
+            <h1>Tunnistetiedot</h1>
+
+            <article>
+              <ContentTable>
+                <tr>
+                  <td>Rekisteritunnus:</td>
+                  <td>M 12345</td>
+                </tr>
+                <tr>
+                  <td>Veneen nimi</td>
+                  <td>Boaty McBoatface</td>
+                </tr>
+                <tr>
+                  <td>Merkki ja Malli:</td>
+                  <td>Yamarin 123</td>
+                </tr>
+              </ContentTable>
+            </article>
+          </>
+        </Col>
+
+        <Delimiter />
```

Comment:
CSS can do the trick here for vertical and horizontal dividers.

Reply:
This component is just a wrapper for css based horizontal and vertical dividers.